modules/installation-overview.adoc

```diff
@@ -22,6 +22,11 @@ Both cluster types have the following characteristics:
 * Highly available infrastructure with no single points of failure is available by default.
 * Administrators maintain control over what updates are applied and when.
 
+The xref:../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc#preparing-to-install-with-agent-based-installer[Agent-based Installer]
```

Comment:
I think we are not allowed to have xrefs in modules. I think you need to add an "Additional resources" section in the assembly right under where you include this module, and list the link in that, per [these instructions](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections).

Reply:
ooops my mistake , yes you are right. Thanks :)